README.md

```diff
@@ -14,8 +14,9 @@ There are two ways you can install command line client:
 - For **Bash** users:
   - Execute `brew install bash-completion` and follow the printed "Caveats" section. 
     <br>Example instruction:<br>
+    Add the following line to your ~/.bash_profile:<br>
```

Comment:
I think you should be able to use verbatim text by aligning backticks under `E`, there shouldn't be a need to use `<br>`.

Reply:
Oh seems that way, since code block spans a new line. Thanks for the catch